packages/tax/src/models/tax-region.ts

```diff
@@ -49,18 +55,21 @@ export default class TaxRegion {
   @Property({ columnType: "text", nullable: true })
   province_code: string | null = null
 
-  @Property({ columnType: "text", nullable: true })
-  parent_id: string | null = null
-
   @ManyToOne(() => TaxRegion, {
     index: "IDX_tax_region_parent_id",
-    cascade: [Cascade.PERSIST],
+    fieldName: "parent_id",
     onDelete: "set null",
```

Comment:
todo: rm, the nullable does that already, also, I suggest to regenerate the migration to validate all the configuration